### PR TITLE
Remove non-public content

### PR DESCRIPTION
**Pull request by stbof**
Partially fixes DOC-5421

### CODE REVIEW

**Comment by andf-crl** [comment]
I am not working with SQL diagrams, but am working with other upstream-generated content. Should this guidance be general to all includes perhaps? If SQL diagram-specific, I don't have any special insight that would make me a good reviewer.

**Comment by stbof** [comment]
> I am not working with SQL diagrams, but am working with other upstream-generated content. Should this guidance be generic to all upstream generated content accessed through includes perhaps? If SQL diagram-specific, I don't have any special insight that would make me a good reviewer.

I understood from @nickvigilante  that you were updating the syntax for remotely including diagrams since it changed from 

`{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ page.version.version | replace: "v", "" }}/grammar_svg/show_stats.html %}`

to 

`
{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ remote_include_version }}/grammar_svg/show_stats.html %}
`
in 22.2.

**Comment by andf-crl** [comment]
> > I am not working with SQL diagrams, but am working with other upstream-generated content. Should this guidance be generic to all upstream generated content accessed through includes perhaps? If SQL diagram-specific, I don't have any special insight that would make me a good reviewer.
> 
> I understood from @nickvigilante that you were updating the syntax for remotely including diagrams since it changed from
> 
> `{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ page.version.version | replace: "v", "" }}/grammar_svg/show_stats.html %}`
> 
> to
> 
> `{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ remote_include_version }}/grammar_svg/show_stats.html %}`
> 
> in 22.2.

No, this is not the case.

File: StyleGuide.md [hunk 1]
```diff
@@ -1045,12 +1045,16 @@ Sometimes, you need to include files that are maintained in other places than th
 - SQL diagrams, which are maintained in the `cockroachdb/cockroach` repo
 - Code samples, which are maintained in various repos
 
-For SQL diagrams, you remotely include the entire (HTML) file as follows:
+###### SQL diagrams
+  
+For [SQL diagrams](https://cockroachlabs.atlassian.net/wiki/spaces/ED/pages/1134166645/SQL+Grammar+Documentation#Add-a-SQL-diagram-to-a-SQL-reference-page), you remotely include the entire (HTML) file as follows:
```
Comment (nickvigilante) [inline StyleGuide.md:1050]: This page requires CRL authentication and is not public. I would consider not hyperlinking to it.

**Review by nickvigilante — CHANGES_REQUESTED** [review]

File: StyleGuide.md (hunk 1 above)
Comment (stbof) [reply StyleGuide.md:1050]: I wonder if we should even include this section at all in the public style guide. Do we expect external users to work with SQL diagrams?

Comment (nickvigilante) [reply StyleGuide.md:1050]: I feel like we won't rely on the public to craft diagrams. We should probably just keep it in our wiki.

File: StyleGuide.md [hunk 2]
```diff
@@ -1021,35 +1021,18 @@ The syntax is a little hard to read inline, but based on some experimenting it a
...
-
-- SQL diagrams, which are maintained in the `cockroachdb/cockroach` repo
-- Code samples, which are maintained in various repos
-
-For SQL diagrams, you remotely include the entire (HTML) file as follows:
-
-```
-{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ page.version.version | replace: "v", "" }}/grammar_svg/show_databases.html %}
-```
+Sometimes, you need to include files that are maintained in other places than the `cockroachdb/docs` repo but referenced in our docs. The `remote_include` tag is used for this. We most often use this tag for code samples, which are maintained in various repos.
```
Comment (nickvigilante) [inline StyleGuide.md:1035]:
```suggestion
Sometimes, you need to include files that are maintained in other places than the `cockroachdb/docs` repo but referenced in our docs. The `remote_include` tag is used for this. We most often use this tag for code samples, which are maintained in various repos controlled by Cockroach Labs.
```

Nit: do you find value in this addition?

**Review by nickvigilante — APPROVED** [review]
LGTM with one nit you can take or leave.

File: StyleGuide.md (hunk 2 above)
Comment (stbof) [reply StyleGuide.md:1035]: I don't think it's aways true. One of the examples `remote_include https://raw.githubusercontent.com/jeffgbutler/mybatis-cockroach-demo/master/src/main/java/com/example/cockroachdemo/CockroachDemoApplication.java` points to this repo: `https://github.com/jeffgbutler/mybatis-cockroach-demo`

**Comment by stbof** [comment]
@nickvigilante  BTW, this section on includes was reviewed quite heavily. I wonder why the link to the wiki was allowed in.

File: StyleGuide.md (hunk 2 above)
Comment (nickvigilante) [reply StyleGuide.md:1035]: Fair. I'll scrap the suggestion. LGTM!

**Comment by nickvigilante** [comment]
@stbof Seems like there are still links to our internal wiki that both Eric and Rich inserted.

File: StyleGuide.md (hunk 1 above)
Comment (rmloveland) [reply StyleGuide.md]: my opinion: 99% of the usage of this doc will be by our team and should be optimized for that use case.  We don't get that many external contributions.  Over the years we have probably spent more time talking about things that could hypothetically affect external contributions than actually reviewing said contributions (they are quite rare)

**Review by rmloveland — APPROVED** [review]
LGTM

Shared my opinions re: internal wiki links in another comment but I don't feel that strongly about it

**Pull request merged**